clean_data.py

```diff
@@ -91,54 +101,34 @@ def sii_from_audiogram(
     return sii.sii(ssl=ssl, nsl=nsl, hearing_threshold=critical_band_hl)
 
 
-def sii_from_df(df, ear='R'):
+def sii_from_df(df, ear):   
```

Comment:
Can you add types to this call?   Ear should be a string, but even better would be to make it a literal, which I just discovered.  That way its clear what the proper values are.  This returns a float, right?

https://stackoverflow.com/questions/64057146/use-type-checking-to-check-that-string-is-of-certain-values-only